examples/pylint/w0133_pointless_exception_statement.py

```diff
@@ -0,0 +1,4 @@
+"""Pointless Exception Statement Example"""
+# The exception statement below is not raised, assigned,
+# or returned for use anywhere in this file.
+Exception("This is a pointless exception statement.")  # Error on this line
```

Comment:
Let's make this example a bit more realistic. Can you try something like the following (but please run pylint/PythonTA to make sure it actually triggers the error!):

```python
def reciprocal(x: float) -> float:
    """Return 1 / x."""
    if x == 0:
        ValueError('x cannot be zero')  # Error on this line
    else:
        return 1 / x
```

and then in the documentation, show how to fix the error.